src/controllers/activity.controller.js

```diff
@@ -0,0 +1,19 @@
+const Activity = require('../models/Activity.model');
```

Comment:
I think the file name is overly complicated - it's in the `controller` folder already so I think you can just call it `activity.js`